kuma/javascript/src/payments/components/subscription-form.jsx

```diff
@@ -10,7 +10,7 @@ import GAProvider, {
 import UserProvider from '../../user-provider.jsx';
 import { getCookie } from '../../utils';
 
-const SUBSCRIPTION_URL = '/api/v1/subscriptions';
+const SUBSCRIPTION_URL = '/api/v1/subscriptions/';
```

Comment:
Hugely important!
Actually, without this fix, it led to a very dangerous problem. 
If you do...
```javascript
fetch('/api/v1/subscriptions', {method:"POST", ...}).then(r => {
  if (response.ok) {
    ....
})
```
what would happen is that you'd see, in the Django server logs:
```
POST /api/v1/subscriptions => 301
GET /api/v1/subscriptions/ => 200
```